src/makeRobot.js

```diff
@@ -37,8 +37,67 @@
  *
  * @return {object}
  */
+
 function makeRobot(name, wheels, version) {
-  // write code here
+  return {
+    name: name,
+    wheels: wheels,
+    version: version,
```

Comment:
if the name of the object's property is the same as its value, you can write it in one word
```suggestion
    name:,
    wheels,
    version,
```